designer/client/src/http/HttpService.ts

```diff
@@ -669,8 +670,12 @@ class HttpService {
         );
         promise
             .then((response) => FileSaver.saveAs(response.data, `${processName}-testData`))
-            .catch((error) =>
-                this.#addError(i18next.t("notification.error.failedToGenerateTestData", "Failed to generate test data"), error, true),
+            .catch((error: AxiosError) =>
+                this.#addError(
+                    i18next.t("notification.error.failedToGenerateTestData", `Failed to generate test data ${handleAxiosError(error)}`),
```

Comment:
Can we use [interpolation](https://www.i18next.com/translation-function/interpolation) for all cases?